migration-guides/v5.0.0.md

```diff
@@ -0,0 +1,21 @@
+# v5.0.0 Migration Guide
+
+_Note: this guide assumes you are upgrading from decanter v4 to v5._
+
+This version contains major updates including the upgrade to Ruby version 3.3.0. Ensure your environment is compatible with this Ruby version before proceeding.
+
+## Major Changes
+
+1. **Ruby Version Update**: Decanter now requires Ruby 3.3.0. This update brings several language improvements and performance enhancements. You must ensure that your environment is running Ruby 3.3.0 or higher. Update your Ruby version using your Ruby version manager (e.g., RVM, rbenv):
```

Comment:
Do you have a link to the summary of improvements & performance enhancements? That'd be nice to include if you've got one handy!